mmseg/models/losses/cross_entropy_loss.py

```diff
@@ -193,13 +255,16 @@ def forward(self,
             class_weight = cls_score.new_tensor(self.class_weight)
         else:
             class_weight = None
+        # Note: In cls_criterion, label < 0 is invalid.
```

Comment:
What is cls_criterion? Here is my suggestion.
```suggestion
        # Note: for BCE loss, label < 0 is invalid.
```